programs/mango-v4/src/state/openbook_v2_market.rs

```diff
@@ -0,0 +1,81 @@
+use anchor_lang::prelude::*;
+use static_assertions::const_assert_eq;
+use std::mem::size_of;
+
+use crate::state::*;
+
+pub type OpenbookV2MarketIndex = u16;
+
+#[account(zero_copy)]
+#[derive(Debug)]
+pub struct OpenbookV2Market {
+    // ABI: Clients rely on this being at offset 8
+    pub group: Pubkey,
+    // ABI: Clients rely on this being at offset 40
+    pub base_token_index: TokenIndex,
+    // ABI: Clients rely on this being at offset 42
+    pub quote_token_index: TokenIndex,
+    pub reduce_only: u8,
+    pub force_close: u8,
+    pub padding1: [u8; 2],
+    pub name: [u8; 16],
+    pub openbook_v2_program: Pubkey,
+    pub openbook_v2_market_external: Pubkey,
+
+    pub market_index: OpenbookV2MarketIndex,
+
+    pub bump: u8,
+
+    pub padding2: [u8; 5],
+
+    pub registration_time: u64,
+
+    pub reserved: [u8; 128],
```

Comment:
recommend bumping this to at least 512 reserved bytes, we'll only have few of these